app/controllers/finance/ecf/change_training_statuses_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Finance
+  module ECF
+    class ChangeTrainingStatusesController < BaseController
+      before_action :participant_profile
```

Comment:
don't think you need this line? it's called in subsequent functions ie `change_training_status_form`